public/scripts/d3/margins.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+
+var width = 1000;
+var height = 1000;
+
+var svg = d3.select("#d3-canvas")
+  .append("svg")
+    // .attr("background-color", "pink")
```

Comment:
I mean commented out code. If you want to merge some commented code into master, then leave a comment to explain why it needs to stay